great_expectations/rule_based_profiler/parameter_builder/simple_date_format_string_parameter_builder.py

```diff
@@ -149,8 +201,17 @@ def _build_parameters(
 
         best_fmt_string: Optional[str] = None
         best_ratio: int = 0
+
+        threshold = get_parameter_value_and_validate_return_type(
+            domain=domain,
+            parameter_reference=self._threshold,
+            expected_return_type=float,
+            variables=variables,
+            parameters=parameters,
+        )
+
         for fmt_string, ratio in format_string_success_ratios.items():
```

Comment:
@NathanFarmer Type hint for fmt_string, ratio please.  Thanks.

Reply:
✅